src/test/java/com/google/sps/servlets/AddSchoolDataTest.java

```diff
@@ -225,36 +171,20 @@ public void AddingNewTerm_NoNewSchool_NewCourse_NoNewProfessor() {
   @Test
   public void AddingNewTerm_NoNewSchool_NewCourse_NewProfessor() {
```

Comment:
Just a nit, but one other thing handy to do in tests is make your variables/asserts/etc follow the same order as the test name (and make all test names follow the same order if you have a bunch of essentially boolean logic).

So in this case, declaring expectedTerm, then school, then course, then professor. Then querying & asserting in that same order.